src/RenderWebGL.js

```diff
@@ -1002,10 +1002,19 @@ class RenderWebGL extends EventEmitter {
      * RenderConstants.ID_NONE if there is no Drawable at that location.
      */
     pick (centerX, centerY, touchWidth, touchHeight, candidateIDs) {
+        const bounds = this.clientSpaceToScratchBounds(centerX, centerY, touchWidth, touchHeight);
+
         candidateIDs = (candidateIDs || this._drawList).filter(id => {
             const drawable = this._allDrawables[id];
+            const drawableBounds = this.getBounds(id);
+            const inRange = (
+                bounds.top < drawableBounds.top
+                    && bounds.right < drawableBounds.right
+                    && bounds.bottom > drawableBounds.bottom
+                    && bounds.left > drawableBounds.left
+            );
```

Comment:
This check could fail in cases like these:
![image](https://user-images.githubusercontent.com/25993062/68529784-22e35b80-02d0-11ea-84fe-e542ce147f44.png)
The black line is the top of `drawableBounds`, and the red line is the top of `bounds`.

It looks like the current check only succeeds if `targetBounds` is completely inside `drawableBounds`.

I would replace this with:
```
const inRange = bounds.intersects(drawableBounds);
```